### ISSUE TITLE
kaleido problem with unsupported format random 'p' format? 

### ISSUE THREAD

**Issue by callumparr** [wdecoster/nanocomp#53]
I am getting a weird error when trying to save out the plots to file in addition to HTML. Either leaving out a format or telling it to save png or pdf it gives this error :

`NanoComp -t 12 --readtype 1D --prefix "F6_interactome_NSC_" --title "F6 interactome ONT-CAGE" --summary $f7 -n NSC_rep1_run1 --outdir NanoComp_output/test --verbose --format png`

or 

`NanoComp -t 12 --readtype 1D --prefix "F6_interactome_NSC_" --title "F6 interactome ONT-CAGE" --summary $f7 -n NSC_rep1_run1 --outdir NanoComp_output/test --verbose` 

```
2022-04-10 23:43:26,611 Nanoget: Finished collecting statistics from summary file /analysisdata/rawseq/fastq/SHARED/000081/Interactome_project_ONT-CAGE/NSC_rep1_run1/guppy_basecaller/sequencing_summary.txt.gz
2022-04-10 23:43:30,069 Reduced DataFrame memory usage from 758.7106475830078Mb to 410.9682674407959Mb
2022-04-10 23:43:39,128 Nanoget: Gathered all metrics of 16574287 reads
2022-04-10 23:43:39,132 Using sequenced read lengths for plotting.
2022-04-10 23:44:04,439 NanoComp: Creating barplots for number of reads and total throughput.
2022-04-10 23:44:06,190 No static plots are saved due to some kaleido problem:
2022-04-10 23:44:06,190 Invalid format 'p'.
    Supported formats: ['png', 'jpg', 'jpeg', 'webp', 'svg', 'pdf', 'eps', 'json']

```

Using NanoComp 1.18, python 3.6 on Debian server.

**Comment by callumparr** [comment]
I've tried going back through some logs of some successful runs and the last time I used and it was running was the version 1.13.1 with python 3.6.7 so I installed this to the new conda environment again using python 3.6.7.

Upon running the similar command I now get a new error.

```
python miniconda3/envs/NanoComp/lib/python3.6/site-packages/nanocomp/NanoComp.py --threads 12 --prefix "F6_interactome_NSC_" --title "F6 interactome ONT-CAGE" --summary /analysisdata/rawseq/fastq/SHARED/000081/Interactome_project_ONT-CAGE/NSC_rep1_run1/guppy_basecaller/sequencing_summary.txt.gz -n NSC_rep1_run1 --outdir NanoComp_output/test --verbose
```


```
2022-04-11 14:31:42,181 NanoComp 1.13.1 started with arguments Namespace(bam=None, barcoded=False, colors=None, cram=None, dpi=100, fasta=None, fastq=None, feather=None, format='png', maxlength=None, minlength=None, names=['NSC_rep1_run1'], outdir='NanoComp_output/test', path='NanoComp_output/test/F6_interactome_NSC_', pickle=None, plot='violin', prefix='F6_interactome_NSC_', raw=False, readtype='1D', split_runs=False, store=False, summary=['/analysisdata/rawseq/fastq/SHARED/000081/Interactome_project_ONT-CAGE/NSC_rep1_run1/guppy_basecaller/sequencing_summary.txt.gz'], threads=12, title='F6 interactome ONT-CAGE', tsv_stats=False, ubam=None, verbose=True)
2022-04-11 14:31:42,181 Python version is: 3.6.7 |Anaconda, Inc.| (default, Oct 23 2018, 19:16:44)  [GCC 7.3.0]
2022-04-11 14:31:42,182 Nanoplotter: valid output format png
2022-04-11 14:31:42,378 Nanoget: Collecting metrics from summary file /analysisdata/rawseq/fastq/SHARED/000081/Interactome_project_ONT-CAGE/NSC_rep1_run1/guppy_basecaller/sequencing_summary.txt.gz for 1D sequencing
2022-04-11 14:32:45,296 Nanoget: Finished collecting statistics from summary file /analysisdata/rawseq/fastq/SHARED/000081/Interactome_project_ONT-CAGE/NSC_rep1_run1/guppy_basecaller/sequencing_summary.txt.gz
2022-04-11 14:32:51,149 Reduced DataFrame memory usage from 758.7106475830078Mb to 410.9682674407959Mb
2022-04-11 14:33:01,454 Nanoget: Gathered all metrics of 16574287 reads
2022-04-11 14:33:01,459 Using sequenced read lengths for plotting.
2022-04-11 14:33:25,858 NanoComp: Creating barplots for number of reads and total throughput.
2022-04-11 14:33:29,896 save() got an unexpected keyword argument 'format'
Traceback (most recent call last):
  File "miniconda3/envs/NanoComp/lib/python3.6/site-packages/nanocomp/NanoComp.py", line 74, in main
    plots = make_plots(datadf, settings)
  File "miniconda3/envs/NanoComp/lib/python3.6/site-packages/nanocomp/NanoComp.py", line 92, in make_plots
    palette=settings["colors"])
  File "/home/callum/miniconda3/envs/NanoComp/lib/python3.6/site-packages/nanocomp/compplots.py", line 94, in output_barplot
    read_count.save(format=figformat)
TypeError: save() got an unexpected keyword argument 'format'
Traceback (most recent call last):
  File "miniconda3/envs/NanoComp/lib/python3.6/site-packages/nanocomp/NanoComp.py", line 235, in <module>
    main()
  File "miniconda3/envs/NanoComp/lib/python3.6/site-packages/nanocomp/NanoComp.py", line 74, in main
    plots = make_plots(datadf, settings)
  File "miniconda3/envs/NanoComp/lib/python3.6/site-packages/nanocomp/NanoComp.py", line 92, in make_plots
    palette=settings["colors"])
  File "/home/callum/miniconda3/envs/NanoComp/lib/python3.6/site-packages/nanocomp/compplots.py", line 94, in output_barplot
    read_count.save(format=figformat)
TypeError: save() got an unexpected keyword argument 'format'

```

**Comment by wdecoster** [comment]
Oh, that's on my, I've been dumb. Let me fix that.

**Comment by wdecoster** [comment]
That's fixed in NanoComp v1.19.0, now on PyPI, and soon on bioconda. The cause was a new feature in NanoPlot: being able to specify multiple output types. And since I didn't think about it long enough this caused a bug in NanoComp. But now also NanoComp supports multiple output types in one go.

**Comment by callumparr** [comment]
> That's fixed in NanoComp v1.19.0, now on PyPI, and soon on bioconda. The cause was a new feature in NanoPlot: being able to specify multiple output types. And since I didn't think about it long enough this caused a bug in NanoComp. But now also NanoComp supports multiple output types in one go.

Thanks so much for the quick fix. brilliant
